<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>699</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Analiza Relacji Biznesowych</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../assets/images.qrc">
    <normaloff>:/images/32.png</normaloff>:/images/32.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab_five">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>22</y>
      <width>650</width>
      <height>600</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>650</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>650</width>
      <height>600</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">float: left;
</string>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab_one">
     <attribute name="title">
      <string>1. Zestawienie</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="text_IloscOsob">
          <property name="text">
           <string>Ilosc osob:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="return_IloscOsob">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="text_IloscMaili">
          <property name="text">
           <string>Ilosc maili:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="return_IloscMaili">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="text_IloscRelacji">
          <property name="text">
           <string>Ilosc relacji:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="return_IloscRelacji">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="text_UzytkownikWyslNajwMaili">
          <property name="text">
           <string>Użytkownik, który wysłał najwiecej maili:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="return_UzytkownikWyslNajwMaili">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="text_UzOdebrNajwMaili">
          <property name="text">
           <string>Użytkownik, który odebrał najwiecej maili:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="return_UzOdebrNajwMaili">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="text_LiczbaMailiForward">
          <property name="text">
           <string>Liczba maili forwardowanych:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="return_LiczbaMailiForward">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="text_DzienWyslanoNajwMaili">
          <property name="text">
           <string>Dzień, w ktorym wysłano najwięcej maili:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="return_DzienWyslanoNajwMaili">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_two">
     <attribute name="title">
      <string>2. Szczegoly</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QTreeWidget" name="treeWidget_MailList">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>Nadawca</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Odbiorca</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Temat</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Data</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_pokazforward">
        <property name="text">
         <string>Pokaz forward</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_four">
     <attribute name="title">
      <string>3. Graf</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QPushButton" name="pushButton_plus">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_minus">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QGraphicsView" name="graphicsView"/>
        <widget class="QWidget" name="gridLayoutWidget_2">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>251</width>
              <height>56</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_five2">
     <attribute name="title">
      <string>4. Filtry</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="pushButton_deleteselectedfilter">
              <property name="text">
               <string>Usun zaznaczony filtr</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_setfiltersaction">
              <property name="text">
               <string>Odfiltruj</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_filters"/>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>TopicSubstringFilter:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QTextEdit" name="textEdit_substringfilterinput">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>46</height>
                 </size>
                </property>
                <property name="html">
                 <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;np. Kon Rafal&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_substringfilter">
                <property name="text">
                 <string>Add TopicSubstringFilter</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>PeopleFilter:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_people"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QTextEdit" name="textEdit_peoplefilterinput">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>39</height>
                 </size>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="html">
                 <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QCheckBox" name="checkBox_issenders">
                  <property name="text">
                   <string>senders</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox_isrecivers">
                  <property name="text">
                   <string>recivers</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_peoplefilter">
                <property name="text">
                 <string>Add PeopleFilter</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>DateFilter:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QDateTimeEdit" name="dateTimeEdit"/>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QCheckBox" name="checkBox_isbefore">
                  <property name="text">
                   <string>przed</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox_isafter">
                  <property name="text">
                   <string>po</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_datefilter">
                <property name="text">
                 <string>Add DateFilter</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_six">
     <attribute name="title">
      <string>5. Zaawansowane filtry</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="textlabel_pobierzemail">
              <property name="text">
               <string>Wybierz sobie email i skopiuj:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_people2"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit_getemail">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="labeltext_wpiszwyrazenie">
            <property name="text">
             <string>Wpisz wyrazenie:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit_advancedfiltertekst">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>150</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="pushButton_advancedfiltersaction">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Action</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_statusadvancedfilters">
          <property name="styleSheet">
           <string notr="true">QLabel { background-color : white;}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="text">
           <string>Jeszcze nic nie zrobiles</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>251</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
     <zorder>labeltext_wpiszwyrazenie</zorder>
     <zorder>plainTextEdit_advancedfiltertekst</zorder>
     <zorder>textlabel_pomocewskazowki</zorder>
     <zorder>textlabel_pobierzemail</zorder>
     <zorder>textEdit_getemail</zorder>
     <zorder>comboBox_people2</zorder>
     <zorder>pushButton_deleteselectedfilter2</zorder>
     <zorder>pushButton_addadvancedfilter</zorder>
     <zorder>listWidget_advancedfilters</zorder>
     <zorder>horizontalSpacer_3</zorder>
     <zorder>horizontalSpacer_4</zorder>
     <zorder>labeltext_wpiszwyrazenie</zorder>
     <zorder></zorder>
     <zorder></zorder>
     <zorder></zorder>
     <zorder>textEdit_pomocewskazowki</zorder>
     <zorder>verticalSpacer_2</zorder>
     <zorder>verticalSpacer_2</zorder>
     <zorder>verticalSpacer_3</zorder>
     <zorder>verticalSpacer_4</zorder>
     <zorder>verticalSpacer_2</zorder>
     <zorder>pushButton_advancedfiltersaction</zorder>
     <zorder>label_statusadvancedfilters</zorder>
    </widget>
    <widget class="QWidget" name="tab_seven">
     <attribute name="title">
      <string>6.Tabela czasowa</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="QListWidget" name="listWidget_people"/>
      </item>
      <item row="0" column="1">
       <layout class="QFormLayout" name="formLayout_personstatistic">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_emptylabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_personfullname">
          <property name="styleSheet">
           <string notr="true">QLabel { background-color : white;}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="text">
           <string>nic</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">QLabel { background-color : white;}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="text">
           <string>Statystyki:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>listWidget_people</zorder>
     <zorder>horizontalSpacer_3</zorder>
     <zorder>formLayoutWidget</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoadFile"/>
   <addaction name="actionReset_2"/>
   <addaction name="actionSettings"/>
  </widget>
  <action name="actionLoadFile">
   <property name="icon">
    <iconset resource="../assets/images.qrc">
     <normaloff>:/images/032.png</normaloff>:/images/032.png</iconset>
   </property>
   <property name="text">
    <string>Zaladuj plik</string>
   </property>
   <property name="toolTip">
    <string>Zaladuj plik</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../assets/images.qrc">
     <normaloff>:/images/015.png</normaloff>:/images/015.png</iconset>
   </property>
   <property name="text">
    <string>Przywroc oryginalne kolory</string>
   </property>
   <property name="toolTip">
    <string>Ustawienia</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../assets/images.qrc">
     <normaloff>:/images/009.png</normaloff>:/images/009.png</iconset>
   </property>
   <property name="text">
    <string>Pomoc</string>
   </property>
   <property name="toolTip">
    <string>Pomoc</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionReset_2">
   <property name="icon">
    <iconset resource="../assets/images.qrc">
     <normaloff>:/images/004.png</normaloff>:/images/004.png</iconset>
   </property>
   <property name="text">
    <string>Resetuj aplikacje</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../assets/images.qrc"/>
 </resources>
 <connections/>
</ui>
